### PR TITLE
Bugfix/admin functions

### PR DESCRIPTION
**Pull request by AnnaWai**

### CODE REVIEW

**Comment by thanh-code-hub** [comment]
you have some conflicts here, if you need help, we can them together sometime this week

**Comment by thanh-code-hub** [comment]
Generally, this looks pretty good. However, we can improve the code style by using of indentations, that would make the code look tidier and easier to read

**Comment by nnduc1994** [comment]
There is a bit of improvement need to be made before merge. i will push the fix

File: app/src/containers/ListOfAdmins/index.js [hunk 1]
```diff
@@ -52,14 +52,23 @@ class ListOfAdmins extends React.Component
...
+        if (value){
+          let time = new Date(value);
+          return +time.getDate()+ '.' +(time.getMonth()+1)+'.' +time.getFullYear()+' '+time.getHours()+'.'+time.getMinutes();
+        }
+        
+        }
     },
     {
-      id: 'userIsActivie',
+      id: 'isActive',
```
Comment (nnduc1994) [inline app/src/containers/ListOfAdmins/index.js:64]: Peter changed the backend data-structure recently. Hence update the property name here

**Review by nnduc1994 — APPROVED** [review]
@bvth  I merge devel, fix conflict push some "safety check" improvement. take a look and approve

**Pull request merged**